Builder/ListBuilder.php

```diff
@@ -164,7 +164,7 @@ public function buildActionFieldDescription(FieldDescriptionInterface $fieldDesc
         }
 
         if (null === $fieldDescription->getType()) {
-            $fieldDescription->setType('action');
+            $fieldDescription->setType('actions');
```

Comment:
Thanks a lot for the due diligence @andrewmy , it makes this feel way safer to merge!